src/components/ui-library/Slider.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import { spacing, Dot } from '../styledComponents';
+
+const styles = {
+  controllers: { marginTop: spacing(5) },
+};
+
+const stringify = (element) => {
+  let res = '';
+  for (let index = 0; index < element.length; index += 1) res += element[index].key;
+  return res;
+};
+
+class Slider extends React.Component {
```

Comment:
Let's try to make all new `Components` functional, rather than class based components - you can use the `useState` hook to use state.